tests/apollo/test_skvbc_checkpoints.py

```diff
@@ -459,7 +459,6 @@ async def test_rvt_conflict_detection_after_corrupting_checkpoint_msg_for_f_plus
                 assert number_of_checkpoint_mismatch == n, \
                     f"Replica {replica_id} number_of_checkpoint_mismatch={number_of_checkpoint_mismatch} != {n}"
 
-    @unittest.skip("Unstable test")
```

Comment:
Why did you enable only a single test?
Maybe I misunderstood you.
We have 4 tests disabled by @StefanPalashev  recently. See:
60e1b8a716f8a8553367059ee4a9de2dccce7dc6
"Disable unstable checkpoint tests"